000-HowToHack/WTH-PRReviewProcess.md

```diff
@@ -0,0 +1,106 @@
+
+1.	Open the Pull Request link in GitHub.
+1.	The policy on the WTH repo is that every PR needs at least **two** “approving reviews” before it can be merged into the WTH repo.
+**NOTE:** To "merge” means to publish the contents/changes contained in the PR into the WTH repo. 
+If you are the first one to start a review on a PR, you will see the following on the right side:
+
+    ![Sample PR Single Reviewer](./images/wth-pr-singlereviewer.png?raw=true "Example PR Single Reviewer")
+1.	You should add a second person to be a reviewer for this PR. To add an additional reviewer, click on the word “Reviewers” as-in the first screenshot below. Then, you may type in the desired reviewer’s name or GitHub handle in the drop down box as demonstrated in the second screen shot below.
+
+    ![Add PR Reviewer - Part 1](./images/wth-pr-addreviewer-01.png?raw=true "Add PR Reviewer 1")
+    ![Add PR Reviewer - Part 2](./images/wth-pr-addreviewer-02.png?raw=true "Add PR Reviewer 2")
+1.	To view the author's content and see it in the web browser, open his/her/their branch by right-clicking on the link to it as shown in the screenshot below. It is a good idea to open it in a separate browser window, side-by-side with the PR itself. 
+![Open PR Content](./images/wth-pr-opencontent.png?raw=true "Open PR Content")
+1.	There are TWO ways to start a “review” of a PR in GitHub. Both require you to navigate to the “Files Changed” tab first. This will show you a list of all the files that were changed and their contents. Red lines represent things changed/removed and the accompanying green lines are what replaced it. 
+![PR Files Changed Tab](./images/wth-pr-fileschangedtab.png?raw=true "PR Files Changed Tab")
+    1. Start a review by making in-line comments – Click the blue “plus” symbol that appears when you mouse over the line number to bring up the comment dialog window.  This is how you will normally start the review as you leave feedback or request changes in specific places in the content. 
+    ![PR In-line Comments](./images/wth-pr-inlinecomments.png?raw=true "PR In-line Comments")
+    1. Start a review from the “Review Changes” button at the top of the “Files Changed” tab.  If you are not requesting any changes or just want to leave a single comment, you would start (and finish) the review this way.
+    ![PR Review Changes Button](./images/wth-pr-reviewchangesbutton.png?raw=true "PR Review Changes Button")
+1.	Do your review!  Follow the guidelines from the [WTH Author’s Guide](https://aka.ms/wthauthor) for help and be on the look out for the common issues:
+    1.	Folder Structure/File organization/Navigation links/etc
+    1.	Links from Student guide to the WTH repo
+    1.	Grammar/spelling/template consistency
+1.	When you are done making in-line comments or you if want to make a only single comment, click the “Review Changes” button at the top of the “Files Changed” tab.  You need to select one of the three options: Comment, Approve, or Request Changes. 
```

Comment:
flipped some words in italics.

"or _if you_ want to make _only a_ single comment"

Reply:
done.